src/FileIo.cpp

```diff
@@ -85,7 +85,9 @@ QString FileIo::pathFromUrl(QString const& _url)
 	if (url.scheme() == "file")
 	{
 		if (path.startsWith("/"))
-			path = path.right(path.length() - 1);
+			path = path.remove(0, 1);
+		if (path.startsWith("/"))
+			path = path.remove(0, 1);
```

Comment:
it was done in purpose,
cause there is in some situation c:/a/b/c
and in some other c:// (a double dash)
this code allows to remove all the leading / in any case.